src/api/app/models/project.rb

```diff
@@ -135,16 +139,8 @@ def cleanup_before_destroy
       end
     end
 
-    # revoke all requests
-    # FIXME: this is breaking request accepting with cleanup, if the project
-    #        will be removed during that
-#    revoke_requests
-  end
-
-  def delete_on_backend(comment = nil)
-    path = source_path
-    path << Suse::Backend.build_query_from_hash({user: User.current.login, comment: comment}, [:user, :comment])
-    Suse::Backend.delete path
+    # revoke all requests that have this project as source/target
+    revoke_requests
```

Comment:
I've just moved this functionality here. It did the same when deleting via API previously....